coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -115,3 +140,23 @@ def get_ignore_comment(self, origin, language):
                         language=language))
                 ignore_comment = None
         return ignore_comment
+
+    def read_ignore_comments(self, original_line, comment_delimiter):
+        """
+        Read Ignore Comment(s) already present in the line and
+        accordingly return a list of bears previously ignored.
+        """
+        if comment_delimiter not in original_line:
+            return
+
+        ignore_regex = re.compile(re.escape(comment_delimiter)
+                                  + r'\sIgnore(\s\S+)+')
+        ignore_comments = [ignore_comment.group() for ignore_comment in
+                           ignore_regex.finditer(original_line.rstrip())]
+        ignored_bears = []
+        bear_regex = re.compile(r'\S+Bear')
```

Comment:
A bear name does not need to end in `Bear`.  See `tests/test_bears/TestBearDep.py`

Reply:
So every bear has a ``Bear`` in it, right? I will make the regex accordingly

Reply:
No, every bear does not have `Bear` in it.  We have functions which get the list of bears which have been loaded.  That list should be used.

Or better, assume all words are bear names here.  I dont think your algorithm here needs to restrict which words are valid bear names.